x-pack/plugins/uptime/public/components/synthetics/step_screenshot_display.tsx

```diff
@@ -11,17 +11,19 @@ import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
 import React, { useContext, useEffect, useRef, useState, FC } from 'react';
 import useIntersection from 'react-use/lib/useIntersection';
-import { UptimeSettingsContext, UptimeThemeContext } from '../../../contexts';
+import { UptimeSettingsContext, UptimeThemeContext } from '../../contexts';
 
 interface StepScreenshotDisplayProps {
+  srcPath?: string;
   screenshotExists?: boolean;
   checkGroup?: string;
   stepIndex?: number;
   stepName?: string;
+  allowPopover?: boolean;
 }
 
-const THUMBNAIL_WIDTH = 320;
-const THUMBNAIL_HEIGHT = 180;
+const THUMBNAIL_WIDTH = 640;
+const THUMBNAIL_HEIGHT = 360;
```

Comment:
The popover is the same size as the thumbnail?

Reply:
have removed this unused code.